test/control-state.html

```diff
@@ -234,6 +234,28 @@
           const instance = document.createElement('test-listeners');
           instance.ready();
         });
+
+        it('should use a fake focus element on tab (firefox)', () => {
+          // Test the workaround for FF63-65 bug that causes the focus to get lost when
```

Comment:
Probably it's better to update the version range in the end to avoid misunderstanding.
The bug shouldn't be reproducible in 63/64

Reply:
It reproduces in 63 also, just tested